k8s/namespaces/idam/idam-user-dashboard/idam-user-dashboard.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-user-dashboard
+  namespace: idam
+spec:
+  releaseName: idam-user-dashboard
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/idam-user-dashboard
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/idam/user-dashboard:prod-* #{"$imagepolicy": "flux-system:idam-user-dashboard"}
```

you need to put a valid image here I think, (may work with invalid with flux v2 but this would break flux v1)
```suggestion
      image: hmctspublic.azurecr.io/idam/user-dashboard:latest #{"$imagepolicy": "flux-system:idam-user-dashboard"}
```

---

Great! Thanks for the suggestion!